extensions/amp-ad-network-adsense-impl/0.1/responsive-state.js

```diff
@@ -64,14 +64,18 @@ export const MAX_HEIGHT_EXP = {
 export class ResponsiveState {
   /**
    * @param {!Element} element
+   * @param {boolean} isContainerWidth
```

Comment:
You will need to mark this arg as optional for type checking, `{boolean=}` or explicitly pass false when applicable.

Reply:
Updated it. Thanks.